README.md

```diff
@@ -37,6 +37,26 @@ Where:
 
 - `$namespace` is optional. It's a string and defaults to 'visualcaptcha'. You'll need to specifically set this if you're using more than one visualCaptcha instance in the same page, so the code can identify from which one is the validation coming from.
 
+### Using Cache
+
+You can use a backend Zend-Cache library options to store images on cache and avoid I/O.
+You'll have to pass options parameter on the cosntructor as document in https://docs.zendframework.com/zend-cache/storage/adapter/.
```

Comment:
Hello it is 0.0.7 on your master so... 
Is it really to downgrade it? 
Or.. do i update it to 0.0.8?
Or.. do i keep ir on 0.0.7?